README.md

```diff
@@ -1,34 +1,41 @@
 
 #### Documentation in other languages
+
 1. [Russian](https://github.com/SasanLabs/VulnerableApp/tree/master/docs/i18n/ru/README.md)
 2. [Chinese](https://github.com/SasanLabs/VulnerableApp/tree/master/docs/i18n/zh-CN/README.md)
 3. [Hindi](https://github.com/SasanLabs/VulnerableApp/tree/master/docs/i18n/hi/README.md)
 4. [Punjabi](https://github.com/SasanLabs/VulnerableApp/tree/master/docs/i18n/pa/README.md)
 
-As Web Applications are becoming very popular these days, there comes the needs to secure them and there are many Vulnerability Scanning Tools but while developing those tools developers need to test those tools, not only that they also need to know how well is the Vulnerability Scanning tool performing but there are no or very less such vulnerable applications exists for testing those tools. There are deliberately vulnerable applications exist in the market but they are not written with such an intent and hence lags extensibility e.g. adding new vulnerabilities is quite difficult.
-
-So generally developer write there own vulnerable applications but that causes productivity loss and also many times rework is done. This Project VulnerableApp is build keeping these factors in mind so this project is scalable, extensible, easiers to integrate and easier to learn.
+As Web Applications are becoming popular these days, there comes a dire need to secure them. Although there are several Vulnerability Scanning Tools, however while developing these tools, developers need to test them. Moreover, they also need to know how well is the Vulnerability Scanning tool performing. As of now, there are little or no such vulnerable applications existing for testing such tools. There are Deliberately Vulnerable Applications existing in the market but they are not written with such an intent and hence lag extensibility, e.g. adding new vulnerabilities is quite difficult. Hence, the developers resort to writing their own vulnerable applications, which usually causes producitivity loss and the pain to rework.
 
-As solving the above issue requires addition of various vulnerabilities, hence it becomes a very good platform to learn various security vulnerabilities. 
+**VulnerableApp** is built keeping these factors in mind. This project is scalable, extensible, easier to integrate and easier to learn.
+As solving the above issue requires addition of various vulnerabilities, hence it becomes a very good platform to learn various security vulnerabilities.
 
 ### Future Goal
-As going further this application might becomes a database for vulnerabilities hence in future it can be used for hosting CTF's and also can be come a compliance for Vulnerability Scanning tools.
 
-## How to use this tool ##
+Going further, this application might becomes a database for vulnerabilities. Hence, in future, it can be used for hosting CTFs and can also become a compliance for Vulnerability Scanning tools.
```

Comment:
I am not sure if we can change compliance to benchmark or make it compliance/benchmark

Reply:
Going ahead with the latter sounds better to me